content/donate.fr.md

```diff
@@ -0,0 +1,58 @@
+
+* <a id="donation-receipt-howto"></a>**Comment obtenir un reçu pour mon don ?**
+
+  Si tu as besoin d'un reçu de notre part pour prouver à ton administration fiscale que tu as effectué un don à un organisme d'utilité publique, nous pouvons t'en fournir un. En règle générale, nous ne délivrons des reçus que pour les paiements effectués par virement bancaire, prélèvement automatique, PayPal ou carte de crédit.
+
+  Après ton don, tu seras redirigé vers une page où tu pourras télécharger un reçu (en allemand) pour ton don si tu as fait un don d'un montant inférieur à 200 €.  
+  Pour un don plus important, tu devras nous envoyer un e-mail à [vorstand@datenanfragen.de](mailto:vorstand@datenanfragen.de), et nous indiquer ton numéro de don et ton adresse, afin que nous puissions te créer un reçu détaillé.
+
+  **Note que nous ne sommes accrédités en tant qu'organisme d'utilité publique que par les autorités fiscales allemandes** Pense à vérifier ton réglement fiscal local pour savoir comment et si tu peux déduire ton don dans tes déclarations fiscales.
+
+* **Qui d'autre a déjà fait des dons ?**
+
+  Les dons des personnes et des entreprises qui ont donné plus de 10% de notre budget annuel sont publiés avec le montant et, seulement avec leur consentement, leur nom. Pour les organisations, nous publierons le nom même sans consentement explicite. Tu peux consulter ces dons dans notre [rapport de transparence]({{< ref "verein/transparency" >}}).
+
+* **Que deviennent les données que j'ai fourni pour faire un don ?**
+
+  En fonction de ta méthode de paiement, tu devras fournir différents types de données. Nous avons cependant passé beaucoup de temps à essayer de réduire au minimum les données que tu dois saisir. Si tu as besoin d'un reçu détaillé pour ton don, nous devrons également récupérer ton adresse et la conserver conformément à la limite de conservation légale. Les données exactes que nous obtenons de toi ou de nos fournisseurs de paiement sont énumérées dans notre [politique de confidentialité]({{< ref "privacy" >}}).
+
+* **Comment puis-je donner de façon anonyme ?**
+
+  Si tu ne souhaites pas indiquer ton identité, tu peux soit utiliser une monnaie cryptée anonyme, soit nous envoyer de l'argent liquide par courrier à notre adresse :
+
+    *Datenanfragen.de e.&thinsp;V.  
+    Schreinerweg 6  
+    38126 Braunschweig  
+    Allemagne*
+
+  Pour les dons en espèces, nous te conseillons d'utiliser le [WERT NATIONAL](https://www.deutschepost.de/de/w/wert-national.html) du service postal allemand ou un produit similaire proposé par ton bureau de poste local. Nous ne recommandons pas d'envoyer des dons en espèces à l'étranger. Dans tous les cas, assure-toi que le contenu de ta lettre n'est pas visible de l'extérieur.
```

Comment:
"Wert National" is an insured and officially sanctioned way of sending cash by mail in German (you usually aren't supposed to). From a quick glance through the La Poste website, they don't seem to offer a similar product.

Unless you know of one, I think we can just get rid of the reference to "Wert National".

Reply:
You're right, in France there is no guarantee on it and no postal service to send cash. I'll remove that.